README.md

```diff
@@ -20,6 +20,16 @@ Basic Usage
 
 See http://mindmup.github.com/editable-table/
 
+
+Config Options
+-----------
+| Name             | Effect                                  | Type    | Default                                                                                                                                                                                                                   |
+|------------------|-----------------------------------------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| arrow_navigation | Disables arrow key navigation on table. | Boolean | ```false```                                                                                                                                                                                                               |
```

Comment:
It's a bit confusing in this way:
- `arrow_navigation == false` means that arrow navigation is enabled
- `arrow_navigation == true` means that arrow navigation is disabled

Maybe naming it `enable_arrow_navigation`, making it default to `true` and inverting all the conditions is clearer